### PR TITLE
[sim] new obstacle interface for 2D sim

### PR DESCRIPTION
**Pull request by danieljiang520**
https://user-images.githubusercontent.com/71047773/170178526-9de686b1-68e1-4fd6-84f0-87f51af3438a.mov

New Obstacle Interface for 2D sim

### CODE REVIEW

File: simulators/nav/src/components/NavSimulator.vue [hunk 1]
```diff
@@ -442,13 +442,13 @@ export default class NavSimulator extends Vue {
...
-        { topic: '/obstacle',       type: 'Obstacle' },
+        { topic: '/obstacle_list',  type: 'ObstacleList' },
         { topic: '/odometry',       type: 'Odometry' },
         { topic: '/rr_drop_init',   type: 'RepeaterDropInit' },
         { topic: '/target_list',    type: 'TargetList' },
         { topic: '/zed_gimbal_cmd', type: 'ZedGimbalPosition' },
-        { topic: '/debugMessage',   type: 'DebugMessage' },
-        { topic: '/projected_points', type: 'ProjectedPoints' }
+        { topic: '/projected_points', type: 'ProjectedPoints' },
+        { topic: '/debugMessage',   type: 'DebugMessage' }
```
Comment (cgiger00) [inline simulators/nav/src/components/NavSimulator.vue:451]: this is probably legacy code you can delete

**Review by cgiger00 — APPROVED** [review]

**Comment by zwgold** [comment]
I notice that we have not changed the LCM for Obstacles to fit what Daniel is sending. (this PR and main still use the bearing/rightBearing/distance struct) Do we want to change that, or keep it as is for now? @cgiger00 @ankithu

**Comment by cgiger00** [comment]
i think keep as is, basically simulator is ahead of nav and perception right now, as long as when you run the sim and obstacle is OFF it doesn't cause the sim the crash. @danieljiang520 if you don't already have a readme plz make one and add this information to it

**Review by zwgold — APPROVED** [review]
has my approval, just add what carmen specified above please

**Review by ankithu — APPROVED** [review]

**Comment by cgiger00** [comment]
**POST CODE FREEZE** previously tested, unrelated to comp

**Pull request merged**